lib/hawkular/alerts/alerts_api.rb

```diff
@@ -81,12 +81,100 @@ def create_trigger(trigger, conditions = [], dampenings = [], _actions = [])
       http_post 'triggers/trigger', full_trigger
     end
 
+    # Creates the group trigger definition.
+    # @param [Trigger] trigger The group trigger to be created
+    # @return [Trigger] The newly created group trigger
+    def create_group_trigger(trigger)
+      ret = http_post 'triggers/groups', trigger.to_h
+      Trigger.new(ret)
+    end
+
+    # Updates a given group trigger definition
+    # @param [String] trigger_id ID of the group trigger to update
+    # @param [Trigger] trigger the group trigger to be updated
+    # @return [Trigger] The updated group trigger
+    def update_group_trigger(trigger_id, trigger)
+      http_put "triggers/groups/#{trigger_id}/", trigger.to_h
+      get_single_trigger trigger_id, false
+    end
+
+    # Creates the group conditions definitions.
+    # @param [String] trigger_id ID of the group trigger to set conditions
+    # @param [String] trigger_mode Mode of the trigger where conditions are attached (:FIRING, :AUTORESOLVE)
+    # @param [GroupConditionsInfo] group_conditions_info the conditions to set into the group trigger with the mapping
+    #                                                    with the data_id members map
+    # @return [Array<Condition>] conditions Array of associated conditions
+    def set_group_conditions(trigger_id, trigger_mode, group_conditions_info)
+      trigger_mode = trigger_mode == :AUTORESOLVE ? 'autoresolve' : 'firing'
+      ret = http_put "triggers/groups/#{trigger_id}/conditions/#{trigger_mode}", group_conditions_info.to_h
+      conditions = []
+      ret.each { |c| conditions.push(Trigger::Condition.new(c)) }
+      conditions
+    end
+
+    # Creates a member trigger
```

Comment:
It could be called create_member_trigger.  @abonas for some general overview on group triggers see http://www.hawkular.org/docs/dev/alerts.html#_group_triggers.  A group trigger is "template-level" and its member triggers are "instance-level".  In short, MIQ Alerts are very much like Group Triggers in H-Alerting.